pavelib/utils/test/utils.py

```diff
@@ -17,7 +17,7 @@ def clean_test_files():
     """
     Clean fixture files used by tests and .pyc files
     """
-    sh("git clean -fqdx test_root/logs test_root/data test_root/staticfiles test_root/uploads")
+    sh("git clean -fqdx test_root/logs test_root/data test_root/uploads")
```

Comment:
Good question. This PR changes Bok Choy servers to use the assets from ``test_root/staticfiles``, rather than picking up the assets from the source tree as devstack usually does. Initially this did break fasttest because ``test_root/staticfiles`` was deleted every time, which meant that the assets were not there when the tests ran. The change in the line of code that this thread is commenting on removed ``test_root/staticfiles`` from the list of folders that are cleaned before each run. This allows ``fasttest`` to keep working (I used it multiple times on this branch). Does that clarify what's happening here?

I'll go make a JIRA story outlining improvements that I'd like to see with fasttest.